### PR TITLE
Ping pong Ball Score Keeper

### PR DESCRIPTION
**Pull request by yellowberard**
This is A pull request for issue number #49 .
@chicken-biryani @Ayushparikh-code  Please review my PR.

### CODE REVIEW

File: Score-Keeper/Score_keeper.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  </div>   
+  <div class="buttons">
+    <button class="p1button">+1 to Player-1</button>
+    <button class="p2button">+1 to Player-2</button>
+    <button class="reset">Reset</button>
+  </div>
+</div>  
+    <script src="Score_keeper.js"></script>
+</body>
+</html>
```
Comment (chicken-biryani) [inline Score-Keeper/Score_keeper.html:46]: Add new line at end of file.

File: Score-Keeper/Score_keeper.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    p1Button.style.background ='#adb5bd';
+    p2Button.style.background='#adb5bd'; 
+ }
+
+//  Limit set by user
+winningScore.addEventListener('change',()=>{
+    limit =parseInt(winningScore.value)
+    reset();
+
+});
```
Comment (chicken-biryani) [inline Score-Keeper/Score_keeper.js:87]: Add a new line at the end of the file. Use a beautifier if needed.

File: Score-Keeper/Score_keeper.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+.winner{
+    color:green;
+   
+}
+
+ .loser{
+     color: red;
+      
+ }
```
Comment (chicken-biryani) [inline Score-Keeper/Score_keeper.css:112]: Use links:

1) Markdown - https://codebeautify.org/markdown-formatter
2) HTML - https://codebeautify.org/htmlviewer
3) CSS - https://codebeautify.org/css-beautify-minify
4) JS -  https://codebeautify.org/jsviewer

**Comment by chicken-biryani** [comment]
Add your project Title and description in index.md

**Comment by yellowberard** [comment]
@chicken-biryani  Done all the necessary changes.

**Pull request merged**